pylace/src/transition.rs

```diff
@@ -108,6 +108,53 @@ impl StateTransition {
     fn __repr__(&self) -> String {
         self.to_string()
     }
+
+    /// Create a dictionary of commonly used state transitions sets.
+    #[staticmethod]
+    fn common_transition_sets() -> std::collections::HashMap<String, Vec<Self>>
```

Comment:
What is the advantage of implementing here, in the pyo3 bindings, instead of in the python code?

Reply:
There is no class within the Python code for the `StateTransition` class; it's entirely within Rust. However, I could easily add a wrapper class for it in Python.

Reply:
I generally want to avoid using magic strings in the Python code, but I suppose it's a typical pattern, so that can be done.

Reply:
Moved into the python code.